### ISSUE TITLE
Version 3.4.17 breaks FontAwesome

### ISSUE THREAD

**Issue by DannyBen** [sass/sass#1808]
Not sure where exactly is the problem since it involves 2-3 different projects.

But, with the recent update to 3.4.17, FontAwesome breaks in production (specifically for me, Heroku - but I _guess_ it breaks in any environment that will precompile).

The resulting CSS simply does not show the icons.

There is an [issue](https://github.com/bokmann/font-awesome-rails/issues/149) also in the FontAwesome repository

Forcing `gem 'sass', '3.4.16'` brings the application back to being operational.

**Comment by chriseppstein** [comment]
Probably a duplicate of #1806.

I'm having a hard time reproducing this.

**Comment by DannyBen** [comment]
I am able to reproduce with a minimal new rails app. Will figure out the reason and/or share the repro case a little later.

**Comment by DannyBen** [comment]
Ok - this is the minimal scss that breaks FontAwesome

```
// app/assets/stylesheets/welcome.scss
@import "font-awesome";
@import url(http://fonts.googleapis.com/earlyaccess/opensanshebrew.css);
```

The result can be seen on http://sassbug.herokuapp.com/

And for convenience, a little screenshot:
![untitled](https://cloud.githubusercontent.com/assets/2405099/9461586/48a1268c-4b18-11e5-988c-f4bc67f05bcd.png)

If you need the full git repository, let me know. It is a rails app built from scratch, with one page and the above stylesheet.

**Comment by mkhairi** [comment]
yeah!.. having same issue here..  i using this material icons. 
`@import url('https://fonts.googleapis.com/icon?family=Material+Icons');`
and it break on production.

**Comment by godfat** [comment]
For people who hit into this, if you're using Rails on Heroku, in addition to downgrade sass to 3.4.16, you'll also need to clear assets cache. In order to clear assets cache, you'll need to install this:
https://github.com/heroku/heroku-repo
For example, you could do:

```
heroku plugins:install https://github.com/heroku/heroku-repo.git
heroku repo:purge_cache --remote heroku # or --app YOUR_APP_NAME
```

Then deploy again. It could take some time to recompile the whole assets.

**Comment by chriseppstein** [comment]
Duplicate of #1806. Fixed in 3.4.18 which was just released.
